docs/installation.adoc

```diff
@@ -0,0 +1,56 @@
+= Installation
+
+== Flink Kubernetes Operator
```

Comment:
do we need both titles if we're just covering installing the operator here?

Reply:
Good point, I removed this title and reduced the level of the others